addons/mail/wizard/mail_compose_message_views.xml

```diff
@@ -79,7 +79,7 @@
                         <button string="Send" attrs="{'invisible': [('is_log', '=', True)]}" name="action_send_mail" type="object" class="btn-primary o_mail_send" data-hotkey="q"/>
                         <button string="Log" attrs="{'invisible': [('is_log', '=', False)]}" name="action_send_mail" type="object" class="btn-primary" data-hotkey="q"/>
                         <button string="Cancel" class="btn-secondary" special="cancel" data-hotkey="z" />
-                        <button icon="fa-lg fa-save" type="object" groups="mail.group_mail_template_editor"
+                        <button icon="fa-lg fa-save" type="object"
```

Comment:
I am not sure to understand this comment :p. 

My proposal is to show this button, instead of hidding it, for regular users :p. 

Regarding
> hide the "Dynamic Placeholder Generator"

You mean adding `groups="mail.group_mail_template_editor"` on this tab ?